caravel/assets/spec/javascripts/sqllab/reducers_spec.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import * as r from '../../../javascripts/SqlLab/reducers'
+import * as actions from '../../../javascripts/SqlLab/actions';
+import { describe, it } from 'mocha';
+import { expect } from 'chai';
+
+describe('sqlLabReducer', () => {
```

Comment:
Couldn't find anything to model these tests off of, let me know how you'd prefer them to be structured.